slyd/app/components/annotations-plugin/component.js

```diff
@@ -584,6 +593,28 @@ export default Ember.Component.extend({
         }
     },
 
+    setIgnores: function() {
```

Comment:
I'm not sure what you mean? Should the array of ignores be accessed through the this.get('pluginState.ignores') as opposed to this.get('ignores')?